geofrontcli/client.py

```diff
@@ -197,7 +197,7 @@ def remotes(self):
                         extra={'user_waiting': False})
             return dict((alias, fmt(remote))
                         for alias, remote in result.items())
-        except:
+        except Exception:
```

Comment:
If I remember correctly, it was intended to catch `KeyboardInterrupt` as well.  It's better catch `BaseException` here IMO.

Reply:
Catching `BaseException` is still harmful because `BaseException` includes non-error exceptions like `StopIteration` or `GeneratorExit`. I think it is better to catch `Exception` and `KeyboardInterrupt` explicitly.

Reply:
I think it's not the case here, because the only thing this `except:` clause does is just logging and reraising.  What's your opinion?

Reply:
In my opinion, it's much better to handle `KeyboardInterrupt` separately instead treating it as an error, since the situation when `KeyboardInterrupt` has been raised represents the obvious intention to abort this execution, not an unexpected failure.